python/arcticdb/vector_db/vector_db.py

```diff
@@ -255,6 +256,19 @@ def _generate_dataframe(
     return df_to_upsert
 
 
+def _process_raw_top_k_result(
+    raw_result: pd.DataFrame,
+    include_vectors: bool,
+) -> List[Dict[str, Union[str, pd.Series]]]:
```

Comment:
I think a list of floats is a more natural return type, as the user doesn't need to know anything about Pandas to use the rest of the API.

Reply:
Hm all right, that makes the test a bit easier.